src/main/java/io/galeb/manager/entity/BalancePolicyType.java

```diff
@@ -54,4 +54,9 @@ public BalancePolicyType setBalancePolicies(Set<BalancePolicy> balancePolicies)
         }
         return this;
     }
+
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?